Projects/Server/Main.cs

```diff
@@ -53,12 +55,21 @@ public static class Core
 
     private static readonly Type[] _serialTypeArray = { typeof(Serial) };
 
+    private static readonly FrozenSet<OSPlatform> UnixPlatforms = 
+        new[] { OSPlatform.Linux, OSPlatform.FreeBSD, OSPlatform.OSX }.ToFrozenSet();
+
     public static readonly bool IsWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
     public static readonly bool IsDarwin = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
     public static readonly bool IsFreeBSD = RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD);
     public static readonly bool IsLinux = RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || IsFreeBSD;
     public static readonly bool IsBSD = IsDarwin || IsFreeBSD;
-    public static readonly bool Unix = IsBSD || IsLinux;
+    public static readonly bool Unix = UnixPlatforms.Any(RuntimeInformation.IsOSPlatform);
+
+    private static readonly SearchValues<string> XUnitAssemblyNames = 
+        SearchValues.Create(["xunit.core", "xunit.runner", "xunit.extensibility", "xunit"], StringComparison.OrdinalIgnoreCase);
+
+    private static readonly ArrayPool<double> CycleArrayPool = ArrayPool<double>.Create(128, 10);
```

Comment:
We can use STArrayPool.Shared here to avoid creating an extra array pool for no reason. This is especially important since MUO is not multithreaded here.